model-optimizer/extensions/front/tf/RetinaNetFilteredDetectionsReplacement.py

```diff
@@ -79,7 +79,7 @@ def append_variances(priors_scale_node: Node, variance: list):
         end.out_port(0).connect(shape_part_for_tiling.in_port(2))
         stride.out_port(0).connect(shape_part_for_tiling.in_port(3))
 
-        concat_value = Const(graph, {'value': np.array([4])}).create_node()
+        concat_value = Const(graph, {'value': int64_array([4])}).create_node()
         shape_concat = Concat(graph, {'name': name + '/shape_for_tiling', 'in_ports_count': 2,
                                       'axis': np.array(0)}).create_node()
```

Comment:
please use `create_op_with_const_inputs` for creation of `Concat` node

Reply:
Agree. But more reasonable is to use `create_op_node_with_second_input`. Done